src/apis/collateral.ts

```diff
@@ -23,4 +27,17 @@ export class DefaultCollateralAPI implements CollateralAPI {
         const account = await this.api.query.dot.account(id);
         return account.free;
     }
+
+    async transferDOT(address: string, amount: string | number): Promise<void> {
+        if (!this.account) {
+            throw new Error("Cannot transfer without account");
+        }
+
+        const transferTx = this.api.tx.dot.transfer(address, amount);
+        await sendLoggedTx(transferTx, this.account, this.api);
+    }
+
+    setAccount(account?: AddressOrPair): void {
```

Comment:
I'm wondering why we made the account parameter optional in all of these setAccount functions